### DISCUSSION TITLE
assests folder automatically filled with preloaded images.

### DISCUSSION THREAD

**Discussion by arunbaby0** [mmistakes/minimal-mistakes#2679] [Q&A] [answered]
I have removed all the unwanted images from the assets folder before doing the bundle serve. Still the images are somehow getting copied into the _site folder. Are there any settings to disable this? And how is this happening?

**Comment by mmistakes** [comment]
The theme doesn't have any images bundled with it so not sure. If you have a public repo I could take a look, otherwise I'm just guessing.

If you forked the theme you likely have files that should be removed. There is content in `/docs` for the documentation site and there is test content in `/test`. After that I would run [Jekyll's clean command](https://jekyllrb.com/docs/usage/) to clear out any cached folders and `_site`.

```
jekyll clean
```

**Comment by arunbaby0** [comment]
I am not sure about this: in gemfile, changing from
gem "minimal-mistakes-jekyll" , path: "../"

to 
gem "minimal-mistakes-jekyll" #, path: "../"

solved my issue

**Comment by mmistakes** [comment, marked as answer]
Looks like you forked the theme... which gives you a `Gemfile` that is meant for developing the theme only.
I do not suggest forking the theme's repo. Instead following the [installation instructions](https://mmistakes.github.io/minimal-mistakes/docs/quick-start-guide/#gem-based-method) by creating a clean `Gemfile`. There are development files in the main repo that will cause you issues if you fork directly.
